Package.swift

```diff
@@ -12,6 +12,7 @@ let package = Package(
     products: [
         .library(
             name: "Introspect",
+            type: .static,
             targets: ["Introspect"]
         )
```

Comment:
I don't think we can accept such a big breaking change. Though the following might be acceptable:

```suggestion
        .library(name: "Introspect", targets: ["Introspect"]),
        .library(name: "Introspect-Static", type: .static, targets: ["Introspect"]),
        .library(name: "Introspect-Dynamic", type: .dynamic, targets: ["Introspect"]),
```

Libraries like [PhoneNumberKit](https://github.com/marmelroy/PhoneNumberKit/blob/master/Package.swift#L10-L12) take the same approach.